proposals/2021-10-community_pool-minimum_deposit/README.md

```diff
@@ -0,0 +1,25 @@
+### Abstract
+
+We would like to change two parameters as follows.
+- Change the community tax from 0% to 2%.
+- Change the minimum proposal deposit from 100,000 MED to 10,000 MED.
+
+
+### Background
+
+Community pool is a means of funding participants who contribute to the Panacea ecosystem.<br/>
+As other Cosmos-based chains, we would like to use the community pool through governance votings to support contributions which boost the Panacea ecosystem.<br/>
```

Comment:
```suggestion
As other Cosmos-based chains, we would like to use the community pool through governance votings to support contributions which boost the Panacea ecosystem, such as outstanding improvements, events and marketing.<br/>
```
What do you think about adding brief examples as above, even though we already posted detail examples in the github issue.

Reply:
Oh, Good!
I think it's better to understand.

Reply:
I had a vague idea that it would be good to put the part you mentioned somewhere.

Reply:
Yeah. For me, either ways is fine if it makes people understand clearly.